shared/js/bluetooth_helper.js

```diff
@@ -240,6 +238,24 @@ var BluetoothHelper = function() {
       });
     },
 
+    isScoConnected: function(cb, errorcb) {
+      _ready(function() {
+        _handleRequest(_adapter.isScoConnected(), cb, errorcb);
+      });
+    },
+
+    sendMediaMetaData: function(metadata, cb, errorcb) {
+      _ready(function() {
+        _handleRequest(_adapter.sendMediaMetaData(metadata), cb, errorcb);
+      });
+    },
+
+    sendMediaPlayStatus: function(status, cb, errorcb) {
```

Comment:
I reference the MDN of `BluetoothAdapter.sendMediaPlayStatus()`. The parameters is also a metadata. I would like to use `metadata`(or `statusMetadata`) instead of `status`. That would be more accurate. Thanks.

[1]: https://developer.mozilla.org/en-US/docs/Web/API/BluetoothAdapter/sendMediaPlayStatus